frontera/contrib/messagebus/zeromq/socket_config.py

```diff
@@ -1,29 +1,33 @@
 # -*- coding: utf-8 -*-
+import socket
 
 
 class SocketConfig(object):
-
-    hostname = None
-    base_port = None
+    __ip_addr = None
```

Comment:
Are you sure we need a [name mangling](http://stackoverflow.com/questions/1301346/the-meaning-of-a-single-and-a-double-underscore-before-an-object-name-in-python) here?  Maybe one underscore, would be the right thing to do?